hadoop-project/pom.xml

```diff
@@ -151,7 +151,7 @@
     <protobuf-maven-plugin.version>0.5.1</protobuf-maven-plugin.version>
     <maven-replacer-plugin.version>1.5.3</maven-replacer-plugin.version>
 
-    <protobuf-compile.version>3.5.1</protobuf-compile.version>
+    <protobuf-compile.version>3.16.3</protobuf-compile.version>
```

Comment:
obsoleted by HADOOP-19065. Update Protocol Buffers installation to 3.21.12 (#6526); needs a version change in hadoop-thirdparty which MUST come out before changing things here.